faker/providers/internet/__init__.py

```diff
@@ -3,10 +3,18 @@
 from __future__ import unicode_literals
 from .. import BaseProvider
 
+from ipaddress import ip_address, ip_network, IPV4LENGTH, IPV6LENGTH
+
 from faker.generator import random
 from faker.providers.lorem.la import Provider as Lorem
 from faker.utils.decorators import slugify, slugify_unicode
 
+try:
+    unicode
+except NameError:  # pragma: no cover
+    unicode = str  # pylint: disable=W0622,C0103
```

Comment:
I don't see the `unicode` module used anytwhere in this diff.

Additionally, `six` is already in the requirements, so if you need this behaviour you can use `six.text_type` https://pythonhosted.org/six/#six.text_type